### PR TITLE
Fixed #35092 -- Exposed extra fields for GeoIP2.country() and GeoIP2.city() responses.

### PR DESCRIPTION
**Pull request by ngnpope**
ticket-35092.

### CODE REVIEW

File: django/contrib/gis/geoip2.py [hunk 1]
```diff
@@ -202,18 +202,23 @@ def city(self, query):
...
             "longitude": response.location.longitude,
+            "metro_code": response.location.metro_code,
             "postal_code": response.postal.code,
-            "region": region.iso_code if region else None,
+            "region_code": region.iso_code if region else None,
+            "region_name": region.name if region else None,
             "time_zone": response.location.time_zone,
+            # Kept for backward compatibility.
+            "dma_code": response.location.metro_code,
+            "region": region.iso_code if region else None,
```
Comment (ngnpope) [inline django/contrib/gis/geoip2.py:221]: I'm thinking we could deprecate these? We'd need to remove these, wrap the dictionary with something to issue a warning, and add a mapping of replacements, i.e. `{"dma_code": "metro_code", "region": "region_code"}`.

I'm just holding off on working on that if it's preferred to just leave these available as above.

Comment (felixxm) [reply django/contrib/gis/geoip2.py:221]: If you want to work on it, I'm not against deprecation but I'm not sure it's worth doing. We can also keep them forever or at least for now.

File: docs/releases/5.1.txt [hunk 2]
```diff
@@ -59,6 +59,14 @@ Minor features
 * :class:`~django.contrib.gis.db.models.Collect` is now supported on MySQL
   8.0.24+.
 
+* The :meth:`~django.contrib.gis.geoip2.GeoIP2.country` result now exposes the
+  ``continent_code``, ``continent_name``, and ``is_in_european_union`` values.
+
+* The :meth:`~django.contrib.gis.geoip2.GeoIP2.city` result now exposes the
```
Comment (felixxm) [inline docs/releases/5.1.txt:65]: `result` sounds unnecessary to me.
```
* The :meth:`.GeoIP2.country` now exposes the ...

* The :meth:`.GeoIP2.city` now exposes the ...
```

**Review by felixxm — COMMENTED** [review]
@ngnpope Thanks :+1: I left small suggestions.

**Comment by ngnpope** [comment]
Thanks @felixxm. Updated.

**Pull request merged**